controllers/selfnoderemediation_controller_test.go

```diff
@@ -272,6 +277,20 @@ func verifyVaDeleted(vaName string) {
 	}, 5*time.Second, 250*time.Millisecond).Should(BeTrue())
 }
 
+func verifyVaNotDeleted(vaName string) {
+	vaKey := client.ObjectKey{
+		Namespace: namespace,
+		Name:      vaName,
+	}
+
+	EventuallyWithOffset(1, func() bool {
```

Comment:
I think when verifying that something isn't deleted, using `Consistently` is a better way than `Eventually`. The latter just checks once and succeeds, when the deletion happens with a short delay. The former will fail as expected.